### PR TITLE
Added resolveToClass method to NameResolver

### PR DESCRIPTION
**Pull request by arothuis**
[Ticket 104934574](https://www.pivotaltracker.com/story/show/104934574)

### CODE REVIEW

**Comment by arothuis** [comment]
@DRvanR @rjkip :calling:

File: src/Surfnet/Conext/EntityVerificationFramework/NameResolver.php [hunk 1]
```diff
@@ -67,4 +73,23 @@ public static function resolveToString($class)
...
         );
     }
+
+    /**
+     * @param string $name
+     * @return string
+     */
+    public static function resolveToClass($name)
+    {
+        Assert::notEmpty($name);
```
Comment (rjkip) [inline src/Surfnet/Conext/EntityVerificationFramework/NameResolver.php]: Nit: notBlank() covers this

File: src/Surfnet/Conext/EntityVerificationFramework/Tests/NameResolverTest.php [hunk 2]
```diff
@@ -97,4 +97,43 @@ public function test_not_implementing_verification_test_cannot_be_converted()
...
+    /**
+     * @test
+     * @group EntityVerificationFramework
+     * @group NameResolver
+     */
+    public function suite_names_are_converted_to_class()
+    {
+        $class = NameResolver::resolveToClass('name_resolver_test_suite');
+
+        $this->assertEquals("Surfnet\\VerificationSuite\\NameResolverTestSuite", $class);
```
Comment (rjkip) [inline src/Surfnet/Conext/EntityVerificationFramework/Tests/NameResolverTest.php]: Should equal `Surfnet\VerificationSuite\NameResolverTestSuite\NameResolverTestSuite`

File: src/Surfnet/Conext/EntityVerificationFramework/Tests/NameResolverTest.php [hunk 3]
```diff
@@ -97,4 +97,43 @@ public function test_not_implementing_verification_test_cannot_be_converted()
...
+     * @group EntityVerificationFramework
+     * @group NameResolver
+     */
+    public function test_names_are_converted_to_class()
+    {
+        $testClass = NameResolver::resolveToClass('name_resolver_test_suite.some_test');
+        $suiteClass = NameResolver::resolveToClass('name_resolver_test_suite.under_scored_test_name');
+
+        $this->assertEquals("Surfnet\\VerificationSuite\\NameResolverTestSuite\\SomeTest", $testClass);
+        $this->assertEquals("Surfnet\\VerificationSuite\\NameResolverTestSuite\\UnderScoredTestName", $suiteClass);
```
Comment (rjkip) [inline src/Surfnet/Conext/EntityVerificationFramework/Tests/NameResolverTest.php]: Should equal `Surfnet\VerificationSuite\NameResolverTestSuite\Test\UnderScoredTestName`. See ticket.

**Comment by arothuis** [comment]
@rjkip @DRvanR

**Comment by rjkip** [comment]
Bar PHPDoc and parameter naming nits, :+1:

**Pull request merged**